watch/migrations/0001_initial.py

```
@@ -0,0 +1,127 @@
+# Generated by Django 6.0.7 on 2026-07-10 21:27
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    initial = True
+
+    dependencies = []
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)